server/middleware/api.coffee

```diff
@@ -113,11 +113,46 @@ putUserSubscription = wrap (req, res) ->
   yield prepaid.redeem(user)
   res.send(user.toObject({req, includedPrivates: INCLUDED_USER_PRIVATE_PROPS, virtuals: true}))
 
+  
+putUserLicense = wrap (req, res) ->
+  user = yield database.getDocFromHandle(req, User)
+  if not user
+    throw new errors.NotFound('User not found.')
+
+  unless req.client._id.equals(user.get('clientCreator'))
+    throw new errors.Forbidden('Must have created the user to perform this action.')
 
+  { ends } = req.body
+  unless ends and DATETIME_REGEX.test(ends)
+    throw new errors.UnprocessableEntity('ends is not a properly formatted.')
+
+  now = new Date().toISOString()
```

Comment:
I would compare using Date#getTime().  I'd assume that the ISO String sort lexically, but longs are cleaner.  Also im paranoid about like Locale stuff going ary.